core/services/pipeline/orm_test.go

```diff
@@ -21,8 +22,29 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+func setupORM(t *testing.T, name string) (db *sqlx.DB, orm pipeline.ORM) {
+	t.Helper()
+
+	if name != "" {
+		_, db = heavyweight.FullTestDB(t, name)
+	} else {
+		db = pgtest.NewSqlxDB(t)
+	}
+	orm = pipeline.NewORM(db, logger.TestLogger(t), cltest.NewTestGeneralConfig(t))
+
+	return
+}
+
+func setupHeavyORM(t *testing.T, name string) (db *sqlx.DB, orm pipeline.ORM) {
```

Comment:
Since there is only one usage of this func, I think it would OK to inline that one special case in order to avoid this extra complexity and indirection :shrug: